tests/integration.rs

```diff
@@ -125,6 +125,7 @@ fn run_in_state() {
 }
 
 #[test]
+#[cfg(feature = "ui")]
```

Comment:
Can you please add `bevy_ui` as a dev dependency and remove the feature flag here? Testing in various configurations is generally a pain.

Reply:
But `ui` is enabled by default. So you will be able to run all tests without changing the configuration.